requests/pandas-profiling.yml

```diff
@@ -0,0 +1,3 @@
+action: archive
+feedstocks:
+  - cf-autotick-bot-test-package
```

Comment:
You need to add the feedstock name here.

Reply:
I really thought I did! My apologies. I've fixed it now.